### ISSUE TITLE
CakeReportPrinter should be disabled in Verbosity = Quiet

### ISSUE THREAD

**Issue by Gargony** [cake-build/cake#2101]
I expect no extra lines from cake-build in Verbosity Quiet mode.

**Comment by patriksvensson** [comment]
@Gargony I would disagree. Setting the verbosity to quiet, simply tells the logger to only output warnings and errors. The report however I think is something that you usually want to see when a build finished even if you want to remove clutter from the log.

We could however introduce a `--no-report` argument to Cake, to tell Cake not to show this.

**Comment by Gargony** [comment]
Thank you for your proposal.
Ok, argument approach also cover my expectations. But in my practices with other command line tools, verbosity applied to all output information (e.g. MSBuild).

**Comment by tapika** [comment]
Any progress on this one ?

**Comment by deniszykov** [comment]
This is kinda bug, not a "feature request". 

Unswitchable printing into `stdout` and cluttering the output is undesired behavior.

**Comment by augustoproiete** [comment]
@deniszykov Are you sending a PR for this one?

**Comment by augustoproiete** [comment]
Possible workaround in the meantime:

```
dotnet cake >/dev/null 2>&1
```

Should suppress all output.

**Comment by deniszykov** [comment]
Sure why not, but it will be straight and simple - verbosity Quiet = no output.

> dotnet cake >/dev/null 2>&1

Also suppress desired output.
